setup/windows setup.md

```diff
@@ -0,0 +1,13 @@
+
+# Windows Setup
+## Java JDK Installation
+1. Download the JDK installer for Windows from [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html):
+2. Run the installer
+3. Add the bin\ directory of the installed JDK to the PATH environment variable, as described [here](http://www.java.com/en/download/help/path.xml)
+4. To verify the JDK installation, open the Command Prompt and type java -version.![](http://i.imgur.com/WdwN6Zc.png) 
+If you have problems installing the JDK, ask for help.
```

Comment:
Looks like this line is wrapped in a weird way near the image. Either put an empty line before it or move it before the image.